README.rst

```diff
@@ -40,7 +40,7 @@ your system:
 
 .. code-block:: bash
 
-    $ wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar -O php-cs-fixer
+    $ wget https://cs.sensiolabs.org/download/php-cs-fixer-v2.phar -O php-cs-fixer
```

Comment:
We might not want to use HTTPS. WDYT?

Reply:
Why would we want to _not_ use HTTPS?

Reply:
well, we wanted HTTPS since beginning, simply https wasn't working properly on this subdomain originally

Reply:
I know, but now it does and I see no reason not to use it, hence my question :)